TLM/TLM/Lifecycle/Patcher.cs

```diff
@@ -83,8 +83,12 @@ public static class Patcher {
         }
 
         public static void Uninstall() {
-            new Harmony(HARMONY_ID).UnpatchAll(HARMONY_ID);
-            Log.Info("TMPE patches uninstalled.");
+            try {
+                new Harmony(HARMONY_ID).UnpatchAll(HARMONY_ID);
+                Log.Info("TMPE patches uninstalled.");
+            } catch(Exception ex) {
+                ex.LogException(true);
```

Comment:
is it suppose to continue after the stuff could not be unpatched as if nothing happened?

Reply:
At least it lets other mods do their thing. in any case, I am logging exception with an on-screen error message.